python/lsst/meas/astrom/anetAstrometry.py

```diff
@@ -48,6 +48,11 @@ class ANetAstrometryConfig(pexConfig.Config):
     rejectIter = pexConfig.RangeField(dtype=int, default=3, doc="Rejection iterations for Wcs fitting",
                                       min=0)
 
+    @property
+    def refObjLoader(self):
+        """An alias, for a uniform interface with the standard AstrometryTask"""
+        return self.solver
+
```

Comment:
Having looked at the code, I agree it's reasonable for now. This change can be reverted when we overhaul the way reference loading is done (e.g. pass in a reference loader to the the astrometry solver in order to support the new index loader).